tests/testthat/test-store.R

```diff
@@ -17,3 +17,66 @@ test_that("file store", {
   st$del(file)
   expect_false(st$exists(file))
 })
+
+test_that("caching file store", {
```

Comment:
How about splitting the test so that each test_that() call tests only one aspect of the behavior?